packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -28,18 +29,36 @@ class DebuggerScreen extends Screen {
 class DebuggerScreenBody extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    if (!kIsWeb) {
+      final theme = Theme.of(context);
+      final textStyle = Theme.of(context)
+          .textTheme
+          .headline
+          .copyWith(color: theme.accentColor);
+      return Center(
+        child: Text(
+          'The debugger screen is only available on web.',
```

Comment:
Add a second line explaining why to avoid any confusion that we thought we were debugging a non web app.
For example:
"Because we have yet to port the debugger to Flutter"

Reply:
Done.